### DISCUSSION TITLE
Qt Ads

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2182] [Media]
https://www.neowin.net/news/ads-may-be-coming-to-kde-the-popular-linux-desktop/

**Comment by tsujan** [comment]
It's disgusting. I hope it's limited to Qt Quick, which we don't use.

EDIT: Yes, it's for Qt Quick :) → https://doc.qt.io/QtDigitalAdvertising/index.html

**Comment by luis-pereira** [comment]
Such a move was more or less expected.
I've nothing against it given that it's use is optional.
